prusti-viper/src/encoder/encoder.rs

```diff
@@ -1680,6 +1687,29 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
         };
         result
     }
+
+    pub fn add_discriminant_info(&self, enum_id: String, discr_id: String, proc_def_id: ProcedureDefId) {
+        let mut map = self.discriminants_info.borrow_mut();
+        let mut previous_entries = map.get_mut(&(proc_def_id.clone(), enum_id.clone()));
```

Comment:
No clone needed?

Reply:
I think they are needed? It doesn't compile without them.